src/app/modules/group/group-routing.module.ts

```diff
@@ -18,6 +18,9 @@ const routes: Routes = [
     children: [
```

Comment:
I found a weird behavior:
When visiting [this page](https://dev.algorea.org/branch/feature/split-personal-data-from-platform-settings/en/#/groups/users/752024252804317630/settings)
Then clicking on "Personal data" works fine
Then clicking on "Progress" I have a weird behavior, all tabs disappear.

As I understand, what we did until today was displaying the allowed tabs and only the focused unallowed tab if there's one.
Here when landing on "Settings" we also display the "Personal data" tab but the access to it is forbidden. I think we should not display the "Personal data" tab
Same, when landing on "Personal data" we also display the "Settings" tab but the access to it is forbidden. I think we should not display the "Settings" tab.

Reply:
Fixed